src/omniscapeImpactTransformer.py

```diff
@@ -14,8 +14,8 @@
 packagesInstalled = mySession.packages()
 omniscapeVersion = packagesInstalled.Version[packagesInstalled.Name == "omniscape"]
 
-if pd.unique(omniscapeVersion)[0] != "1.1.2":
-    sys.exit("The omniscapeImpact add-on package version 1.0.2 requires the omniscape base package version 1.1.2.")
+if pd.unique(omniscapeVersion)[0] != "2.1.1":
```

Comment:
Hm, I think we can delete this bit of code. SyncroSim will already check to make sure the specified version of omniscape is loaded from the `builtFor` tag in the package.xml